### ISSUE TITLE
UX: new-button to return to landing screen really confusing

### ISSUE THREAD

**Issue by creynders** [docker/kitematic#1973]
### Expected behavior

Being able to return to the landing screen, after having selected a container in the "containers" section

### Actual behavior

No button to be found, until you realise it's hidden under the new-button in the "containers" section

### Information about the Issue

Probably at some point in time it made sense to return to the opening screen with "new", but ATM "new" doesn't really fit the bill anymore, since there's more on the landing screen than just the ability to create a new container, but also browsing through "my images", "my repos" etc.
Granted, all of these _do_ allow you to create a new container, but they also allow you to do more.

**Comment by FrenchBen** [comment]
@creynders The idea is that you are trying to create a new container when going through that page. 
Perhaps it should be labelled `Dashboard` although really you can't do much more than create a new container on that page, even if they are from your image or your own repos.

If you have a design in mind, please do share.

**Comment by creynders** [comment]
@FrenchBen yeah, strictly speaking "new" _is_ correct. But I do suppose many people, like myself, will go looking for that images overview screen and won't find it.
Ideally, it would be something like this, I think:

![kitematic-ui](https://cloud.githubusercontent.com/assets/436347/18308325/527bc9bc-74f4-11e6-8d04-207a6ed08b1f.jpg)

I.e. the search, filter, my repos and my images would be permanently visible

**Comment by FrenchBen** [comment]
@elesant WDYT?
